benchmark/CMakeLists.txt

```diff
@@ -5,18 +5,35 @@ target_compile_options(waf_benchmark PRIVATE $<$<BOOL:${LIBDDWAF_ENABLE_LTO}>:-f
 target_link_libraries(waf_benchmark PRIVATE libddwaf_objects lib_yamlcpp lib_rapidjson m)
 target_include_directories(waf_benchmark PRIVATE ${libddwaf_SOURCE_DIR}/src)
 
-set_target_properties(waf_benchmark PROPERTIES
-    CXX_STANDARD 20
-    CXX_STANDARD_REQUIRED YES
-    CXX_EXTENSIONS NO)
+if (LIBDDWAF_BENCHMARK_PGO_STAGE1)
+    set_target_properties(waf_benchmark PROPERTIES
+        CXX_STANDARD 20
+        CXX_STANDARD_REQUIRED YES
+        CXX_EXTENSIONS NO
```

Comment:
you can set these common properties just once

Reply:
True, I keep mindlessly copying this everywhere